README.md

```diff
@@ -12,8 +12,15 @@
 
 ### Dependencies
 
-gtkcord4 needs GTK4, gobject-introspection, and optionally libcanberra. If compiling, then the library
-headers are also required.
+Gtkcord4 needs the following dependencies met:
+
+```
+golang git gtk4-devel gobject-introspection gobject-introspection-devel
```

Comment:
`golang` -> `go`

Reply:
Also, what distro are these packages for? Why are there `-devel` packages?

Reply:
On Fedora and RHEL (and likely others), the `-devel` suffix is used instead of `-dev`. 
These are needed at compile time for the library headers iirc.

Reply:
Can we have a separate section for Fedora packages then? This seems to be pertaining to just Fedora.

Also, since so many people insist, I think it'd be better to just list out all the dependencies on every distro that people care enough about to PR in.